src/authority/gnd.js

```diff
@@ -100,6 +103,18 @@ export class GND extends Registry {
         if (json.professionOrOccupation && json.professionOrOccupation.length > 0) {
           output.profession = json.professionOrOccupation.map((prof) => prof.label);
         }
+        if (json.preferredNameEntityForThePerson && json.preferredNameEntityForThePerson.length > 0) {
+          json.preferredNameEntityForThePerson.map((fullName) => {
```

Comment:
nit: the return of the map is not used. can be a forEach.

Same with the other `map`

Reply:
Also, the function is marked as async, but the `await` keyword is not used. I'd take a second and rewrite it to async/await which is more readable imho.